tests/conftest.py

```diff
@@ -43,6 +43,23 @@ def seed_dynamic_data(request, app, clear_test_data, _db, enable_preserve_test_d
     _db.session.commit()
 
 
+@pytest.fixture(scope="function")
+def clean_db(app, _db):
+    """Ensures a clean database before each test runs"""
```

Comment:
can we verify is seed_dynamic_data also doing it or not ?

Reply:
This was verified as part of my work on the ticket - `seed_dynamic_data` does not clear the database beforehand which means our test would be unstable using this, because we are checking that there are exactly two `Lizt` items in the database